langchain/src/vectorstores/prisma.ts

```diff
@@ -276,6 +294,18 @@ export class PrismaVectorStore<
     return results;
   }
 
+  buildSqlFilterStr(filter: PrismaSqlFilter) {
+    return Object.keys(filter)
```

Comment:
_note_: Not sure whether that would be possible, but as Prisma reexports `sql-template-tag`, it might be more sane to defer the joining logic to that instead?